meta-mentor-staging/recipes-core/meta/nativesdk-sdk-relocate.bb

```diff
@@ -35,6 +35,7 @@ sed -i -e "s#\$DEFAULT_INSTALL_DIR\>#\$installdir#g" "\$env_setup_script"
 END
     cat "${COREBASE}/meta/files/toolchain-shar-relocate.sh" >>sdk-relocate
     sed -i -e 's#grep -v "\$target_sdk_dir/.*"#grep -Ev "(sdk-relocate|$target_sdk_dir/(environment-setup-*|\.installpath))"#' sdk-relocate
+    sed -i -e '/native_sysroot=/a native_sysroot=$($SUDO_EXEC echo $native_sysroot | sed -e "s:\\$scriptdir:$target_sdk_dir:")' sdk-relocate
```

Comment:
I have verified this change with SDK. Added the following line in local.conf
`
TOOLCHAIN_HOST_TASK_append = " nativesdk-sdk-relocate nativesdk-qtbase-tools nativesdk-qtbase-tools-dev "
`
Later built populate_sdk. And ran the populate_sdk script on the host and below is the result:
`
sujith@kdekidd0:~/MEL/dogwood/build-mx6q/tmp/deploy/sdk$ sudo ./mel-glibc-x86_64-core-image-sato-cortexa9hf-neon-sdk-next+snapshot.sh
[sudo] password for sujith: 
Mentor Embedded Linux SDK installer version next+snapshot
=========================================================
Enter target directory for SDK (default: /opt/mel/next+snapshot): 
You are about to install the SDK to "/opt/mel/next+snapshot". Proceed[Y/n]? 
Extracting SDK...............................................................................................................................................................done
Setting it up...find: `$scriptdir/sysroots/x86_64-melsdk-linux/lib': No such file or directory
SDK could not be set up. Relocate script unable to find ld-linux.so. Abort!
sujith@kdekidd0:~/MEL/dogwood/build-mx6q/tmp/deploy/sdk$ cd /opt/mel/next+snapshot
sujith@kdekidd0:/opt/mel/next+snapshot$ pwd
/opt/mel/next+snapshot
sujith@kdekidd0:/opt/mel/next+snapshot$ ls -l
total 56
-rw-r--r-- 1 root root   842 Dec 22 20:54 cb-mbs-options
-rw-r--r-- 1 root root  4529 Jan  2 10:40 environment-setup-cortexa9hf-neon-mel-linux-gnueabi
-rwxrwxr-x 1 root root  9071 Dec 30 11:34 relocate_sdk.py
-rw-r--r-- 1 root root 23032 Dec 22 20:54 site-config-cortexa9hf-neon-mel-linux-gnueabi
drwxr-xr-x 4 root root  4096 Dec 30 10:12 sysroots
-rw-r--r-- 1 root root   137 Dec 22 20:54 version-cortexa9hf-neon-mel-linux-gnueabi
sujith@kdekidd0:/opt/mel/next+snapshot$ ls sysroots/x86_64-melsdk-linux/usr/bin/
qt5  sdk-auto-relocate  sdk-relocate
sujith@kdekidd0:/opt/mel/next+snapshot$ ls sysroots/x86_64-melsdk-linux/usr/bin/qt5/
fixqt4headers.pl  moc  qdbuscpp2xml  qdbusxml2cpp  qlalr  qmake  rcc  syncqt  syncqt.pl  uic
sujith@kdekidd0:/opt/mel/next+snapshot$ ./sysroots/x86_64-melsdk-linux/usr/bin/qt5/qmake -v
QMake version 3.0
Using Qt version 5.6.2 in /opt/mel/next+snapshot/sysroots/x86_64-melsdk-linux/usr/lib
sujith@kdekidd0:/opt/mel/next+snapshot$ 
`